x-pack/plugins/threat_intelligence/public/modules/indicators/hooks/use_aggregated_indicators.ts

```diff
@@ -65,6 +67,8 @@ export interface RawAggregatedIndicatorsResponse {
 }
 
 export const useAggregatedIndicators = ({
+  filterQuery,
```

Comment:
Does it make sense to get these values from `useFilters` hook instead of passing it down through multiple components and then to the hook?

Reply:
I guess I didn't really think about it, I just mimicked the way it's done in the useIndicators (for the table)

Reply:
my thinking as we are not using `filterQuery` and `filters` for any logic in `IndicatorsBarChartWrapper` rather than passing them through to this hook, it might make sense to call the useFilters hook directly here and get this data.

Reply:
got it! It should be better now. Sorry it took me a bit longer than expected I struggled fixed a storybook issue, had to use a story `decorator` with the React `MemoryRouter` (had never done that before)